src/common/intl/locales/fr.js

```diff
@@ -768,8 +767,8 @@ export default {
   // Original text: "Remote"
   editBackupRemoteTitle: 'Emplacement',
 
-  // Original text: 'Delete the old backups first'
-  deleteOldBackupsFirst: undefined,
+  // Original text: "Delete the old backups first"
+  deleteOldBackupsFirst: 'Supprimez les anciennes sauvegardes',
```

Comment:
It's a title in the Disaster Recovery creation form, with this description: "Delete the old backups before copy the vms. If the backup fails, you will lose your old backups."
So it would be *Supprime**r*** instead of *Supprime**z***.

Any other modification?